README.md

```diff
@@ -71,8 +71,10 @@ deproxy server, and workload tests should use wrk client and nginx server.
 ## Requirements
 
 - Host for testing framework: `Python2`, `python2-paramiko`,
-`python-configparser`, `python-subprocess32`, `wrk`, `ab`, `python-scapy`,
-`python-cryptography`, `scapy-ssl_tls` (installed with `pip`), `h2spec`
+`python-configparser`, `python-subprocess32`, `wrk`, `ab`, `scapy`
+(Debian 10 contains outdated 2.4.0 and we require 2.4.4, so install it with pip),
```

Comment:
File `check_deps/check_python_dependencies.py` contains list of python modules that must be installed using system package manager. And `scapy` is  still listed there. Need to remove t from there. 

In the same time, there is more pythonic way to install missing modules from `requirements.txt`, so all the modules required to be installed from `pip` should be listed there.

Reply:
Added requirements.txt. Do you think we need to update README?

Reply:
the `modules_apt` list in `check_deps/check_python_dependencies.p` still has a `scapy` member, please remove it from there. We have a `check_deps` directory with scripts to install all the required modules and programs. Previously we relied mostly on disrto repositories, and never called `pip` there. Probably we need to add `pip` call into `check_deps/check_framework.sh`.

About Readme update. I don't think that any updates are really required. The file just states which modules are required and for which needs. So user can manually check all the dependencies and manually install them. While `check_deps` was targeted to roll the test environment out of box.